mathesar/templates/mathesar/index.html

```diff
@@ -17,6 +19,18 @@
     {% endfor %}
   {% endif %}
 
+  {% comment %} Making it async to start the execution as soon as its downloaded without making it wait for the HTML to be parsed {% endcomment %}
+  <script type="module"  async>
+  var isDevelopmentMode = {{ development_mode|yesno:"true,false" }}
+    var src = isDevelopmentMode ? "{{ client_dev_url }}/src/i18n/{{ preferred_language }}/index.ts" : "{% static manifest_data.module_translations_file %}"
```

Comment:
This logic would be better hidden away from the user, if we place them in the template as python code, instead of JS.